easybuild/easyconfigs/s/spglib-python/spglib-python-2.1.0-foss-2023a.eb

```
@@ -0,0 +1,29 @@
+easyblock = 'PythonBundle'
+
+name = 'spglib-python'
+version = '2.1.0'
+
+homepage = 'https://pypi.python.org/pypi/spglib'
+description = "Spglib for Python. Spglib is a library for finding and handling crystal symmetries written in C."
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('scikit-build-core', '0.5.0'),
+]
```

`scikit-build-core` is only a build dependency.

```suggestion
builddependencies = [
    ('scikit-build-core', '0.5.0'),
]

dependencies = [
    ('Python', '3.11.3'),
    ('SciPy-bundle', '2023.07'),
]
```